test/controllers/licences.controller.test.js

```diff
@@ -403,9 +403,15 @@ describe('Licences controller', () => {
       }
     })
 
-    describe('when a request is valid and has returns', () => {
+    describe('when a request is valid and has returns and the licence has requirements', () => {
```

Comment:
We are fine with just this test. The controller tests are there to confirm we have the route and controller hooked up correctly in Hapi and that they respond with the status codes we expect.

This is why you'll more often see in our tests just

```javascript
describe('when a request is valid', () => {
  // ...
})

describe('when a request is invalid', () => {
  // ...
})
```

```suggestion
    describe('when a request is valid', () => {
```